create_reference_tsv.py

```diff
@@ -0,0 +1,38 @@
+import sys
+inFile = sys.argv[1]
+outFile = sys.argv[2]
+
+sampleDict = {}
+
+with open(inFile, 'r') as f:
+	lines = f.readlines()
+
+k = 0
+for i in lines:
+	sample = i.split('/')[-1].split('_')[0]
+	readnr = i.split('/')[-1].split('_')[3]
+	if sample in sampleDict and readnr == "R1":
+		sampleDict[sample] = sampleDict[sample] + 1
+		if k != len(lines)-1:
+                	nextread = lines[k+1].split('/')[-1].split('_')[3]
+		if nextread == "R2":
+			newline = '\t'.join((sample,"ZZ","0",sample,str(sampleDict[sample]),i.rstrip("\n"),lines[k+1]))
```

Comment:
I'm not entirely sure if there is a "right" way, but I think the 5th field will be used as a read tag in e.g. the BAM files. Therefore it could be used to hold some information (in addition to separating the reads). In the WGS tsv, this is a string with `"[FCID].[laneno].[ssheet_idx]"`.

The FCID could be extracted from the fastq file path.

Reply:
I can not find anything about this in the documentation. But if we think that this feature is missing it is fairly easy to add in a new PR.

Reply:
This is a good reference for how GATK uses this information and what it should contain: https://gatk.broadinstitute.org/hc/en-us/articles/360035890671-Read-groups

If you look at the Sarek code here: https://github.com/nf-core/sarek/blob/2.7/main.nf#L1184, you can see that Sarek assigns the same value (equal to the 5th field above) for `ID` and `PU`, which is why I think it would be good to format this according to above.